src/RouteJs.AspNet/router.js

```diff
@@ -60,7 +60,7 @@
 	
 	////////////////////////////////////////////////////////////////////////////////////////////////
 	
-	var Route = function (route) {
+	var Route = function (route, routeManager) {
```

Comment:
Instead of passing in the whole route manager, what about just passing in a settings object? The object could just have the `lowerCaseUrls` property.